config/config.go

```diff
@@ -1,19 +1,24 @@
 package config
 
 import (
+	"time"
+
 	"github.com/kelseyhightower/envconfig"
 )
 
 // Config represents the configuration required for the dp-download-service
 type Config struct {
-	BindAddr           string `envconfig:"BIND_ADDR"`
-	EncryptionDisabled bool   `envconfig:"ENCRYPTION_DISABLED"`
-	PrivateKey         string `envconfig:"RSA_PRIVATE_KEY"`
-	DatasetAPIURL      string `envconfig:"DATASET_API_URL"`
-	DatasetAuthToken   string `envconfig:"DATASET_AUTH_TOKEN"`
-	SecretKey          string `envconfig:"SECRET_KEY"`
-	VaultToken         string `envconfig:"VAULT_TOKEN"`
-	VaultAddress       string `envconfig:"VAULT_ADDR"`
+	BindAddr                string        `envconfig:"BIND_ADDR"`
+	BucketName              string        `envconfig:"BUCKET_NAME"`
+	DatasetAPIURL           string        `envconfig:"DATASET_API_URL"`
+	XDownloadServiceToken   string        `envconfig:"DOWNLOAD_SERVICE_TOKEN"     json:"-"`
```

Comment:
Why does this var start with X?

Reply:
I think because the http header begins with an `X` because it is a custom header, but agree it may be more readable without it!